soci/soci_index.go

```diff
@@ -255,7 +253,7 @@ func buildSociLayer(ctx context.Context, cs content.Store, desc ocispec.Descript
 		Digest:         ztocDesc.Digest.String(),
 		OriginalDigest: desc.Digest.String(),
 		Type:           ArtifactEntryTypeLayer,
-		Location:       path.Join(config.SociIndexDirectory, desc.Digest.String()),
+		Location:       desc.Digest.String(),
```

Comment:
Should we just remove location now?

Reply:
I'm not sure about that, since it will require changes to the artifacts.db schema which will be breaking change. I can submit another PR to do the same.